api/job/v1beta1/job.proto

```diff
@@ -63,19 +63,13 @@ message GetAvailableJobsRequest {
   // Should be used consistently across all APIs.
   string area_key = 1;
   string worker_id = 2;
-  double range_limit = 3;
+  int32 precision = 3;
   int32 limit = 4;
 }
 
-message Distance {
-  api.type.v1beta1.Location from_location = 1;
-  string unit = 2;
-  float range = 3;
-}
-
 message AvailableJob {
   JobInfo job = 1;
-  Distance distance = 2;
+  int32 distance = 2;
```

Comment:
comment that says it's in meters

Reply:
maybe better to rename the property to distance_meters. ? @kevinmichaelchen